recipes/pupnp/all/conandata.yml

```diff
@@ -1,4 +1,4 @@
 sources:
-  "1.12.1":
-    url: https://github.com/pupnp/pupnp/archive/release-1.12.1.tar.gz
-    sha256: b4fc8f8e78d5d3259f321aecab1bc9d9ef7b9cea816c4b4d850b264120773290
+  "1.14.2":
+    url: https://github.com/pupnp/pupnp/archive/release-1.14.2.tar.gz
+    sha256: e1487800425e47a26998593c42256a61caa1cd1ec210029fe8050cf6bab9c334
```

Comment:
it is not allowed to remove an existing version

Reply:
Even if the version has CVE?

Reply:
question to ask to @uilianries @SSE4 and @danimtb

Reply:
@uilianries @SSE4 @danimtb, any comment? I did search but there is nothing relevant that I could find.

In e.g. Linux distros packages with known vulnerability are generally removed from the repo.

Reply:
we have to discuss the policy regarding this. Thanks a lot for the ping

Reply:
Any news @danimtb?

Reply:
Hi @whyman,
We are not removing binaries from ConanCenter binary repository and we try to keep this repo in sync for the moment, so we will not be removing the version affected from the conandata.yml. Please create a new entry for the new version. Of course, patches for declared CVEs will be welcomed!